spec/features/create_mediated_page_request_spec.rb

```diff
@@ -7,9 +7,11 @@
       visit new_mediated_page_path(item_id: '1234', origin: 'SPEC-COLL', origin_location: 'STACKS')
       click_link "I don't have a SUNet ID"
 
+      expect(page).to have_field('Library ID', type: 'text')
       expect(page).to have_field('Name', type: 'text')
       expect(page).to have_field('Email', type: 'email')
       expect(page).to have_css('a', text: '‹ Go back (show the login option)')
+      expect(page).to have_css('input#mediated_page_user_attributes_library_id', :focus)
```

Comment:
> On May 26, 2015, at 11:29 PM, Jessie Keck <notifications@github.com> wrote:
> 
> In spec/features/create_mediated_page_request_spec.rb <https://github.com/sul-dlss/sul-requests/pull/154#discussion_r31106379>:
> 
> >        expect(page).to have_field('Name', type: 'text')
> >        expect(page).to have_field('Email', type: 'email')
> >        expect(page).to have_css('a', text: '‹ Go back (show the login option)')
> > +      expect(page).to have_css('input#mediated_page_user_attributes_library_id', :focus)
> Try commenting our the javascript that applies the focus and see if this test still passes. I have a feeling it will pass regardless because :focus is not an appropriate filter for have_css.
> 

Yes, that does pass the test regardless.  

I tried `have_css('input#mediated_page_user_attributes_library_id:focus’)` but there were no matches. 

I also tried `expect(page.evaluate_script('$("input#mediated_page_user_attributes_library_id").is(":focus")')).to be_true` but the script evaluates to false. 

I cannot seem to find any test that will detect whether the field is focused, which makes me wonder if the test is actually doing or registering the field focus.
> —
> Reply to this email directly or view it on GitHub <https://github.com/sul-dlss/sul-requests/pull/154/files#r31106379>.
>